tool_quality_checks/test/data/instellingen_mini_schiedam.ini

```diff
@@ -20,4 +20,5 @@ min_dekking=0.5
 hoogte_verschil=0.2
 min_dimensions=0.2
 max_dimensions=5
-padding_manhole=0.2
\ No newline at end of file
+padding_manhole=0.2
+max_connections=8
```

Comment:
Je editor kan je denk ik wel instellen op "eindig altijd met een newline" ofzo? Standaard voor programmeurs om dat ingesteld te hebben.
(In dit geval was de `padding_manhole` regel dan niet aangepast omdat er geen enter achter hoefde).